### ISSUE TITLE
feature_request(version): latest version for git:changelog

### ISSUE THREAD

**Issue by Kristinita** [release-it/release-it#541]
[**Same issue for release-it 6.1.1**](https://github.com/release-it/release-it/issues/218).

### 1. Summary

It would be nice, if [**generate-changelog**](https://www.npmjs.com/package/generate-changelog) will show correct version in GitHub Releases page.

![GitHub Releases](https://i.imgur.com/qWODbVo.png)

#### 2.1. Data

+ Part of [**my `.release-it.yaml`**](https://github.com/Kristinita/SashaReleaseItTesting/blob/8776c2628d3ed0952a650ce7477bb70766c8b3c7/.release-it.yaml):

```yaml
after:bump:
    changelog
git:
    changelog: changelog -f -
```

[**generate-changelog Commands**](https://www.npmjs.com/package/generate-changelog#cli):

+ `changelog` — append to top of `CHANGELOG.md` changelog of last version,
+ `changelog -f -` — changelog of last version to stdout.

#### 2.2. Steps to reproduce

1. I [**release 0.2.2 version**](https://github.com/Kristinita/SashaReleaseItTesting/releases/tag/st3-0.2.2).
1. I add, commit and push some changes.
1. `release-it minor -n -V`.

#### 3.1. Expected

[**Correct version in `CHANGELOG.md`**](https://github.com/Kristinita/SashaReleaseItTesting/commit/c8b27ddaa2514eebc76bb661a05c7a2e4762cb7a#diff-4ac32a78649ca5bdd8e0ba38b7006a1e):

![CHANGELOG.md version](https://i.imgur.com/aAaEk03.png)

#### 3.2. Non-expected behavior

[**Not correct version in GitHub Releases**](https://i.imgur.com/qWODbVo.png):

![GitHub Releases](https://i.imgur.com/qWODbVo.png)

#### 4.1. safeBump

[**`safeBump`**](https://github.com/release-it/release-it/issues/218#issuecomment-360466964) from version 9 is [**deprecated option**](https://github.com/release-it/release-it/commit/35659648cd9193794cf70a2c3151fab669ca6c76). I can't find in documentation, how I can replace it.

#### 4.2. release-it documentation

I read [**this**](https://github.com/release-it/release-it/blob/master/docs/changelog.md) docs page, but I can't find, how I can solve my problem.

#### 4.3. generate-changelog

generate-changelog [**can't add specific version**](https://www.npmjs.com/package/generate-changelog#cli) as `changelog --release-version ${version}`.

#### 5.1. Use auto-changelog

I can't find, how I can generate `CHANGELOG.md` via [**auto-changelog**](https://www.npmjs.com/package/auto-changelog) in the same format as generate-changelog.

### 6. Environment

+ Windows 10 Enterprise LTSB 64-bit EN
+ Node.js 12.7.0
+ git 2.22.0.windows.1
+ generate-changelog 1.8.0
+ release-it 12.3.5

Thanks.

**Comment by webpro** [comment]
The `changelog` tool I haven't used before and is indeed not documented at all. Would be nice if we could add this to the docs, if someone could get it to work nicely with release-it. You can use `DEBUG=release-it:* release-it [...]` to see exactly what's going on during the process and what/when custom/third-party scripts are executed.

By the way, the `safeBump` option has been removed (deprecated in v9, removed in v10). It's no longer relevant, as this functionality (create/update changelog) is moved to plugin(s).

**Comment by Kristinita** [comment]
Yes, this issue still relevant. #FuckStaleBot.

Thanks.

**Comment by webpro** [comment]
Closing this issue, as I'm personally not interested in using this 3rd party tool. Feel free to open a PR documenting how to use generate-changelog, I'm happy to help out and merge this.
